atc/atccmd/command.go

```diff
@@ -1447,6 +1458,18 @@ func (cmd *RunCommand) constructLockConn(driverName string) (*sql.DB, error) {
 	return dbConn, nil
 }
 
+func (cmd *RunCommand) parseUnsafeOverrides() (atc.UnsafeWorkerOverrides, error) {
+	for fromPath, toPath := range cmd.Unsafe.MountHolepunches {
+		if !filepath.IsAbs(fromPath) {
+			return atc.UnsafeWorkerOverrides{}, fmt.Errorf("mount-holepunch source must be an an absolute path")
+		}
+		if !filepath.IsAbs(toPath) {
```

Comment:
👍 to this being absolute - I think it technically *could* be relative and effectively mount in to each task's working directory, but I could see that causing conflicts with regular build inputs